d2l-typography-shared-styles.html

```diff
@@ -3,7 +3,8 @@
 <dom-module id="d2l-typography-shared-styles">
```

Comment:
I am wondering if it is still necessary to wrap these mixins in Polymer's `dom-module`?  If not, consumers would not need to specify it in their `include`, but rather just `import` this HTML.

```
<link rel="import" href="../polymer/polymer.html">
<style is="custom-style">
	html {
		--d2l-body-standard-text: { ... };
	}
</style>
```